### PR TITLE
Set up backend

### LINKED ISSUE #3: Set up backend

### PR DESCRIPTION
**Pull request by ReedOnly**
closes #3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,23 +1,30 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+# Backend
+backend/*/bin
+backend/*/obj
+backend/scripts/InitQuestions.json
```
Comment (petterwildhagen) [inline .gitignore:4]: Delete this line?

File: .gitignore [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
+# Editors
+.vscode
+.idea
 
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
+#Cypress
+frontend/cypress/videos
+frontend/cypress/screenshots
```
Comment (petterwildhagen) [inline .gitignore:30]: Do we need these folders for Cypress? We dont have them for BMT, videos and screenshot are really only relevant for CI, and they get uploaded there.

Comment (ReedOnly) [reply .gitignore:30]: yes, but they do get created in this folder if you run it locally! we have the same gitignore in BMT

File: .gitignore (hunk 1 above)
Comment (ReedOnly) [reply .gitignore]: fixed

**Review by petterwildhagen — APPROVED** [review]

**Pull request merged**